mysql-database-id/3.initialize-database.sql

```diff
@@ -0,0 +1,19 @@
+USE eventuate;
+
+CREATE TABLE new_message (
+  id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
The issue with this schema is that the type of the 'ID' column is changed.
As a result, it's no longer compatible with the older clients, which makes upgrading more complicated:  the library upgrade and the code changes need to be done together and perhaps the CDC too - not sure.

A better approach would be for the `AUTO_INCREMENT` PK to have a different name so the old `ID` column (now NULLable and not a PK, obviously) and the new PK can be in the same table.

Reply:
Chris, ok, I will be working in that direction.